recipes/mender-client/debian-master/rules

```diff
@@ -7,7 +7,7 @@ override_dh_auto_build:
 	make CGO_CFLAGS="${CGO_CFLAGS}" CGO_LDFLAGS="${CGO_LDFLAGS}" CC="${CC}" \
 	GOOS=linux CGO_ENABLED=1 build
 
-install_targets_exclude=install-systemd
+install_targets_exclude=install-systemd install-modules-gen
```

Comment:
Just to be clear: We are not now removing the generator scripts from a package which we expect users to install on their build machine, right? Like `mender-client.deb` on your x86_64 desktop machine? From the documentation it looks like we expect people to download the scripts manually, but I'm just making sure.

Reply:
We don't recommend, document nor expect users to have installed `mender-client` on a workstation (iow not device that needs updates). Someone could have done this and we will break his/her flow but I think it is unlikely.

A different topic is if we should package the scripts somewhere else (`mender-artifact-generators`?) and encourage users to install them on their machines. Basically: `apt-get install mender-artifact mender-artifact-generators` and good to go.

In any case, not in this PR :sweat_smile: